docs/pages/reference/backends.mdx

```diff
@@ -620,6 +620,282 @@ Config option priority is applied in the following order:
+                "s3:ListMultipartUploadParts",
+                "s3:GetObjectVersion",
+                "s3:GetObject",
+                "s3:DeleteObjectVersion",
+                "s3:DeleteObject",
+                "s3:AbortMultipartUpload"
+            ],
+            "Effect": "Allow",
+            "Resource": [
+                "arn:aws:s3:::audit-transient/results/*",
+                "arn:aws:s3:::audit-transient/large_payloads/*",
+                "arn:aws:s3:::audit-long-term/events/*"
+            ],
+            "Sid": "AllowMultipartAndObjectAccess"
+        },
+        {
+            "Action": "sns:Publish",
+            "Effect": "Allow",
+            "Resource": "arn:aws:sns:eu-central-1:1234567890:audit-sqs",
+            "Sid": "AllowPublishSNS"
+        },
+        {
+            "Action": [
+                "sqs:ReceiveMessage",
+                "sqs:DeleteMessage"
+            ],
+            "Effect": "Allow",
+            "Resource": "arn:aws:sqs:eu-central-1:1234567890:audit-sns",
+            "Sid": "AllowReceiveSQS"
+        },
+        {
+            "Action": [
+                "glue:GetTable",
+                "athena:StartQueryExecution",
+                "athena:GetQueryResults",
+                "athena:GetQueryExecution"
+            ],
+            "Effect": "Allow",
+            "Resource": [
+                "arn:aws:glue:eu-central-1:1234567890:table/audit_db/audit_table",
+                "arn:aws:glue:eu-central-1:1234567890:database/audit_db",
+                "arn:aws:glue:eu-central-1:1234567890:catalog",
+                "arn:aws:athena:eu-central-1:1234567890:workgroup/audit_workgroup"
+            ],
+            "Sid": "AllowAthenaQuery"
+        },
+        {
+            "Action": [
+                "kms:GenerateDataKey",
+                "kms:Decrypt"
+            ],
+            "Effect": "Allow",
+            "Resource": "arn:aws:kms:eu-central-1:1234567890:key/kms_id",
+            "Sid": "AllowAthenaKMSUsage"
+        }
+    ]
+}
+```
+
+### Migration from Dynamo to Athena audit logs backend
+
+<Admonition
+  type="tip"
+  title="Tip"
+>
+  Migration is only needed if Dynamo audit logs was used and you want to keep
+  old data.
+</Admonition>
+
+Migration consist of following steps:
+
+1. Set up Athena infrastructure
+1. Dual write to both Dynamo and Athena, query from Dynamo
+1. Migrate old data from Dynamo to Athena
+1. Dual write to both Dynamo and Athena, query from Athena
+1. Disable writing to Dynamo
+
+In teleport storage config, `audit_events_uri` accepts multiple URLs used as
+audit loggers. If more than 1 is used, then events are written to each audit
+system, and queries are executed from first one.
+
+<Admonition
+  type="tip"
+  title="Tip"
+>
+  If anything goes wrong during migration steps 1-4, rollback to Dynamo solution
+  is as trivial as making sure it's on first place inside `audit_events_uri`
+  and removing Athena URL from `audit_events_uri`.
```

Comment:
```suggestion
  If anything goes wrong during migration steps 1-4, roll back to the Amazon DynamoDB solution by making sure its URL is the first value in the `audit_events_uri` field
  and removing the Athena URL.
```
- Suggestion for clarity
- Removing "as trivial as" since, if a user needs to read documentation to do something, it's probably not trivial